docs/usage.md

```diff
@@ -2,55 +2,55 @@
-      * [Nextflow memory requirements](#nextflow-memory-requirements)
-  * [Other command line parameters](#other-command-line-parameters)
-    * [`--outdir`](#--outdir)
-    * [`--email`](#--email)
-    * [`--email_on_fail`](#--email_on_fail)
-    * [`--max_multiqc_email_size`](#--max_multiqc_email_size)
-    * [`-name`](#-name)
-    * [`-resume`](#-resume)
-    * [`-c`](#-c-1)
-    * [`--custom_config_version`](#--custom_config_version)
-    * [`--custom_config_base`](#--custom_config_base)
+- [nf-core/scrnaseq: Usage](#nf-corescrnaseq-usage)
+  - [Table of contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Running the pipeline](#running-the-pipeline)
+    - [Updating the pipeline](#updating-the-pipeline)
+    - [Reproducibility](#reproducibility)
+  - [Core Nextflow arguments](#core-nextflow-arguments)
+    - [`-profile`](#-profile)
+    - [`--reads`](#--reads)
+    - [`--aligner` (Required)](#--aligner-required)
+      - [Alevin](#alevin)
+        - [`--salmon_index`](#--salmon_index)
+        - [`--txp2gene`](#--txp2gene)
+      - [STARSolo](#starsolo)
+        - [`--star_index`](#--star_index)
+      - [Kallisto | BUStools](#kallisto--bustools)
+        - [`--bustools_correct`](#--bustools_correct)
+        - [`--skip_bustools`](#--skip_bustools)
+        - [`--kallisto_gene_map`](#--kallisto_gene_map)
+        - [`--kallisto_index`](#--kallisto_index)
+    - [Cellular barcodes](#cellular-barcodes)
+      - [`--type` to specify droplet type (Required)](#--type-to-specify-droplet-type-required)
+      - [`--chemistry` (using cellranger barcodes) (Required)](#--chemistry-using-cellranger-barcodes-required)
+      - [`--barcode_whitelist` for custom barcode whitelist](#--barcode_whitelist-for-custom-barcode-whitelist)
+  - [Reference genomes](#reference-genomes)
+    - [`--genome` (using iGenomes)](#--genome-using-igenomes)
+    - [`-c`](#-c)
+      - [Custom resource requests](#custom-resource-requests)
+    - [`--fasta`](#--fasta)
+    - [`--gtf`](#--gtf)
+    - [`--transcript_fasta`](#--transcript_fasta)
+    - [`--save_reference`](#--save_reference)
+    - [`--igenomes_ignore`](#--igenomes_ignore)
+  - [Job resources](#job-resources)
+    - [Automatic resubmission](#automatic-resubmission)
+    - [Custom resource requests](#custom-resource-requests-1)
+  - [AWS Batch specific parameters](#aws-batch-specific-parameters)
+    - [Running in the background](#running-in-the-background)
+      - [Nextflow memory requirements](#nextflow-memory-requirements)
+  - [Other command line parameters](#other-command-line-parameters)
+    - [`--outdir`](#--outdir)
+    - [`--email`](#--email)
+    - [`--email_on_fail`](#--email_on_fail)
+    - [`--max_multiqc_email_size`](#--max_multiqc_email_size)
+    - [`-name`](#-name)
+    - [`-resume`](#-resume)
+    - [`-c`](#-c-1)
+    - [`--custom_config_version`](#--custom_config_version)
+    - [`--custom_config_base`](#--custom_config_base)
```

Comment:
```suggestion
```

Reply:
The table of contents should be deleted in `usage.md` as it is created automatically on the website